### DISCUSSION TITLE
Is it possible to know the combo cards?

### DISCUSSION THREAD

**Discussion by AltRepoYT** [masterking32/MasterHamsterKombatBot#119] [Q&A]
Is it possible to know the combo cards, regardless of whether we have the cards or want to buy them? Similar to how we can obtain a Cipher code immediately at reset time, can we at least know what the combo cards are, regardless of our intention to buy them or have them?

**Comment by miladesign** [comment]
https://api21.datavibe.top/api/GetCombo

but its update with >3 hour delay

**Reply by AltRepoYT** [reply]
> https://api21.datavibe.top/api/GetCombo
> 
> but its update with >3 hour delay

I was looking for something faster just to know what the cards are, just like the cipher code. Three hours is a lot of delay; I mean, you could get it faster on the web, but thank you regardless.

**Comment by Hzone-glitch** [comment]
> https://api21.datavibe.top/api/GetCombo
> 
> but its update with >3 hour delay

It show wrong card "medium"
![Screenshot_2024-08-31-14-27-52-008_com.android.chrome.jpg](https://github.com/user-attachments/assets/78319c61-bf9d-4ecb-b401-a49cffbca466)

Can you fix this?

**Reply by Fy0urM** [reply]
> > https://api21.datavibe.top/api/GetCombo
> > but its update with >3 hour delay
> 
> It show wrong card "medium" ![Screenshot_2024-08-31-14-27-52-008_com.android.chrome.jpg](https://private-user-images.githubusercontent.com/52880633/363408683-78319c61-bf9d-4ecb-b401-a49cffbca466.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUxMDI2NTMsIm5iZiI6MTcyNTEwMjM1MywicGF0aCI6Ii81Mjg4MDYzMy8zNjM0MDg2ODMtNzgzMTljNjEtYmY5ZC00ZWNiLWI0MDEtYTQ5Y2ZmYmNhNDY2LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODMxVDExMDU1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVhNTFhMjhkZGYyMDUwYzI1NWViYzk5YmQwZGI5YmI5Y2M4NTcwZmU2ODJmM2RiZDUwMjllM2I1NTAyZjQyNWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hxcB52u0TJxEQCGFxyyOBe98HeRIs-6YrTp-GV2269w)
> 
> Can you fix this?

This api is filled out by the author, it has nothing to do with the game, you can't get information about combos from the game until you buy the card involved in the combo

**Comment by tboy1337** [comment]
Duplicate of https://github.com/masterking32/MasterHamsterKombatBot/issues/69
